guide/sections/wis2node.adoc

```diff
@@ -16,9 +22,11 @@ TODO: to be completed
 
 ==== Connecting with Global Services
 
-This section describes the process by which a WIS node is registered with one or more Global Broker and Global Cache components.
+This section describes the process by which a WIS2 node is registered with one or more Global Broker and Global Cache components.
 
-TODO: to be completed
+A WIS2 node shall allow at least one Global Broker and Global Cache to connect to its local broker and web storage. For this, the WIS2 node shall allow the outgoing IP addresses used by the Global Broker and Global Cache to establish connections to broker and web-storage, and allow the Global Cache to subscribe to the topic corresponding to this WIS2 node using the username and password combination "everyone/everyone".
```

Comment:
At least two of each... Redundancy.
No, Global Cache don't subscribe to WIS2 Node. Only to GB. 
No one but GBs are subscribing to WIS2 Nodes, GC and GBs.

Replace by:

A WIS2 node shall allow at least two Global Broker to connect to its local broker and at least two Global Cache to its web storage. For this, the WIS2 node shall allow the outgoing IP addresses used by the Global Broker and Global Cache to establish connections to broker and web-storage, and allow the Global Broker to subscribe to the topic corresponding to this WIS2 node. Username and password used to protect the MQTT subscription will be part of the Global Registry.